modules/virt-about-upgrading-virt.adoc

```diff
@@ -37,3 +37,13 @@ used to manage the virtual machine process.
 
 * DataVolumes and their associated PersistentVolumeClaims are preserved during
 upgrade.
++
+[IMPORTANT]
+====
+Running virtual machines that cannot be live migrated might block an {product-title} cluster upgrade.
+This includes virtual machines that use hostpath-provisioner storage or SR-IOV network interfaces.
+
+As a workaround, you can reconfigure the virtual machines so that they can be powered off during a cluster upgrade. Remove the `evictionStrategy: LiveMigrate` field and
```

Comment:
Should we say "powered off automatically" ?

Reply:
Edited per your comment